cfme/common/provider_views.py

```diff
@@ -176,6 +177,13 @@ class ProvidersManagePoliciesView(BaseLoggedInPage):
     """
     policies = VersionPick({Version.lowest(): DynaTree('protect_treebox'),
                             '5.7': BootstrapTreeview('protectbox')})
+
+    @View.nested
+    class entities(BaseNonInteractiveEntitiesView):  # noqa
+        @property
+        def entity_class(self):
+            return ProviderEntity().pick(self.browser.product_version)
```

Comment:
I think this property and verpick malarkey will be resolved by https://github.com/RedHatQE/widgetastic.core/issues/59